src/Controllers/BatchProcessorController.ts

```diff
@@ -482,6 +492,108 @@ module StreamStats.Controllers {
                 this.deleteBatch(batchID, deleteCode, batchStatusEmail);                
             }
         }
+        
+        // submit batch job
+        public submitBatch(submit250:boolean=false): void {

```

Comment:
When a batch is submitting, can we have a "wait" toaster appear? We typically show these toasts after submitting requests that take a while:

`this.toaster.pop("wait", "Submitting Batch", "Please wait...", 0);`

Example from when a user delineates a basin:
![image](https://github.com/USGS-WiM/StreamStats/assets/40237491/e376a6a6-cf33-480b-9227-8ac1676b62ad)